shakespeare.cabal

```diff
@@ -49,6 +49,9 @@ library
                    , blaze-html
                    , exceptions
                    , transformers
+                   , vector
+                   , unordered-containers
+                   , scientific
```

Comment:
I don't believe these extra dependencies should be considered a problem at all, since they are already transitive dependencies via aeson.